web-animations-2/Overview.bs

```diff
@@ -2006,6 +2045,39 @@ partial interface AnimationTimeline {
 
 </div>
 
+<h3 id="the-animation-interface">The <code>Animation</code> interface</h3>
+
+Update the startTime and currentTime of the Animation interface as follows:
+
+<pre class="idl">
+[Exposed=Window]
+partial interface Animation : EventTarget {
+    attribute CSSNumberish?       startTime;
+    attribute CSSNumberish?       currentTime;
+};
+</pre>
+
+Replace [=start time=] and [=current time=] attribute descriptions with:
+
+<div class='attributes'>
+ :   <dfn attribute for=Animation>startTime</dfn>
+ ::  Update the description as:
+>    The [=start time=] of this animation. When associated with a
```

Comment:
Since this is nested in a list this requires more indentation, The '>' needs to be aligned with Update on the line above. There's an example with delay and endDelay below, e.g.
```
 ::  Update the description as:
     > The [=start time=] of this animation. When associated with a
     > [=progress-based timeline=], [=start time=] must be returned as a
     > {{CSSNumericValue}} with percent units. Otherwise [=start time=] must be
```

Reply:
The list as a whole is indented incorrectly - for some reason it has a 1-space indent before each item, which doesn't do anything but *does* make the subsequent 4-space indents also misaligned. It *should* be indented one level from the container `div`, and then yeah, the blockquote would need two levels to nest it under the `dd` properly.

Just use tab indents; humans can't reliably space-indent to save their lives. ^_^

Reply:
I don't have much experience with tab indenting specs but I'll take your word that it's simpler. I guess it takes away the temptation to use anything less than a full tab indent ^_^. I take it I should update https://github.com/w3c/csswg-drafts/pull/6565#pullrequestreview-744230939 or follow up with another PR to do this for the whole spec and then we can update this to match.